sphinx/writers/html5.py

```diff
@@ -325,7 +325,7 @@ def add_permalink_ref(self, node, title):
             format = u'<a class="headerlink" href="#%s" title="%s">%s</a>'
             self.body.append(format % (node['ids'][0], title, self.permalink_text))
 
-    def generate_targets_for_listing(self, node):
+    def generate_targets_for_table(self, node):
```

Comment:
From this on, the purpose of the method will be changed.
I agree for that, but you need to update the docstring. This is no longer used for listings. And the target version of docutils will be changed.